### ISSUE TITLE
Bestbuy antibot issues

### ISSUE THREAD

**Issue by HieuDGits** [zeldridge/best-buy-sniper#33]
Bestbuy usually ups it’s antibot system during really hyped drops like 30 serie cards.

1. There is a queue-like system for carting which someone pointed out already.
2. An account is required to purchase any graphics card
3. After getting to the payment page they require you to do phone or email verification which you then need to enter the code they send.

The first two issues seem like easy fixes and someone already has a bandaid fix for issue 1 in one of the comments. Issue 3 seems to be the most challenging of the bunch. This is what i know now from my experience from manually checking out a 3060 ti. I will try to update you guys on my progress.

**Comment by Rushan4eg** [comment]
Make it please...hoping on you

**Comment by KingKline** [comment]
I added queuing system,

I made it sign in instead of just be a guest (it also makes the bot faster because no need to type payment info)

For the sms and email checks, it would be hard to give out to people because I made it just linked to my info

I will give the first two tho just add me on discord KingKline#7295

**Comment by kingofdeath09** [comment]
just wondering of where the solution would be posted? Thanks.

**Comment by atomicultimatum** [comment]
Any chance @KingKline you have found a fix or could be done with finding a fix before Tuesday?

**Comment by atomicultimatum** [comment]
Also, do you know why is it doing this?
<img width="960" alt="Screenshot 2021-03-26 235322" src="https://user-images.githubusercontent.com/77267580/112712767-9d455900-8e8e-11eb-8145-234ea4c67445.png">

**Comment by Spareo** [comment]
> I added queuing system,
> 
> I made it sign in instead of just be a guest (it also makes the bot faster because no need to type payment info)
> 
> For the sms and email checks, it would be hard to give out to people because I made it just linked to my info
> 
> I will give the first two tho just add me on discord --------

Mind sharing the code to do the login and purchase while logged in?

**Comment by atomicultimatum** [comment]
what's your discord?

**Comment by bshah0625** [comment]
@KingKline I tried to implement my knowledge of python and attempted to by pass the queuing system but I am not sure if it will work since I tried to use the '1' popping up on the cart logo as the indicator for the item being added to the cart thus meaning it has by passed the queuing system. Plus you have implemented a sign in method. I would appreciate if I would be able to get your code from you. 

Appreciate it if you could send me your discord or I can send you mine.

Thanks in advance!

**Comment by atomicultimatum** [comment]
I hope he answers today cause we need it before tomorrow

**Comment by blitzshooter** [comment]
Please add me on discord or to the discords or whatever. Trying to get this bot up and running so I can *maybe* get a 3060ti. Maxwhale#8874 @Spareo @KingKline

**Comment by atomicultimatum** [comment]
Yeah plz cause the one big drop that's not going to happen again for a while has a very high chance of dropping tomorrow between 6-9 PST/9-12 EST. If you guys can add me and send what you have, my discord is Daniel Swyers#9212. Plz and thank you @Spareo @KingKline @bshah0625, especially the issue 3 solve @KingKline.
